README.md

```diff
@@ -108,20 +108,20 @@ Arguments to pass:
 
 Description: Triggers an event in the simulator
 
-## Runing SimConnect on other system.
+## Running SimConnect on another system.
```

Comment:
perhaps these would be better?
... on a separate system.
or
... on a different system.